tests/acceptance/features/bootstrap/CliContext.php

```diff
@@ -86,6 +86,20 @@ public function theAdministratorResetsThePasswordOfUserUsingTheCLI(string $user,
 		$this->featureContext->updateUserPassword($user, $password);
 	}
 
+	/**
+	 * @When the administrator checks the backup consistency using the CLI
+	 *
+	 * @return void
+	 */
+	public function theAdministratorChecksTheBackupConsistencyUsingTheCli():void {
+		$path = $this->featureContext->getStorageUserOcisRoot();
+		$command = "backup consistency -p $path -b ocis";
```

Comment:
`ocis` is the default one
```suggestion
		$command = "backup consistency -p $path";
```